### PR TITLE
feat(content): blutwurst recipe

### PR DESCRIPTION
**Pull request by chrsrns**
## Purpose of change

If #4727 gets merged, this will add a blood sausage comestible to incentivize bleeding of corpses.

### CODE REVIEW

File: data/json/items/comestibles/meat_dishes.json [hunk 1]
```diff
@@ -223,6 +223,43 @@
     "description": "Lean sausage made from salt-cured offal, that has been cooked.",
     "flags": [ "EATEN_HOT" ]
   },
+  {
+    "id": "sausage_blood",
+    "copy-from": "sausage_raw",
```
Comment (Lamandus) [inline data/json/items/comestibles/meat_dishes.json:228]: to get "better results" try to copy from sausage instead.

```suggestion
    "copy-from": "sausage",
```

File: data/json/recipes/food/meat.json [hunk 2]
```diff
@@ -1613,6 +1613,90 @@
...
+  {
+    "type": "recipe",
+    "result": "sausage_blood",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 4,
+    "charges": 2,
+    "time": "40 m",
+    "autolearn": true,
```
Comment (Lamandus) [inline data/json/recipes/food/meat.json:1625]:
```suggestion
    "autolearn": true,
    "book_learn": [ [ "family_cookbook", 2 ], [ "scots_cookbook", 2 ] ],
```

File: data/json/recipes/food/meat.json [hunk 3]
```diff
@@ -1613,6 +1613,90 @@
...
+    "type": "recipe",
+    "result": "sausage_blood",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "id_suffix": "large",
+    "difficulty": 4,
+    "charges": 4,
+    "time": "40 m",
+    "autolearn": true,
```
Comment (Lamandus) [inline data/json/recipes/food/meat.json:1656]:
```suggestion
    "autolearn": true,
    "book_learn": [ [ "family_cookbook", 2 ], [ "scots_cookbook", 2 ] ],
```

File: data/json/recipes/food/meat.json [hunk 4]
```diff
@@ -1613,6 +1613,90 @@
...
+    "type": "recipe",
+    "result": "sausage_blood",
+    "id_suffix": "smoke",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 4,
+    "charges": 1,
+    "time": "30 m",
+    "autolearn": true,
```
Comment (Lamandus) [inline data/json/recipes/food/meat.json:1694]:
```suggestion
    "autolearn": true,
    "book_learn": [ [ "family_cookbook", 2 ], [ "scots_cookbook", 2 ] ],
```

File: data/json/recipes/food/meat.json [hunk 5]
```diff
@@ -1677,6 +1761,72 @@
...
+  {
+    "type": "recipe",
+    "result": "sausage_blood_raw",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "difficulty": 4,
+    "charges": 2,
+    "time": "10 m",
+    "autolearn": true,
```
Comment (Lamandus) [inline data/json/recipes/food/meat.json:1773]:
```suggestion
    "autolearn": true,
    "book_learn": [ [ "family_cookbook", 2 ], [ "scots_cookbook", 2 ] ],
```

File: data/json/recipes/food/meat.json [hunk 6]
```diff
@@ -1677,6 +1761,72 @@
...
+    "type": "recipe",
+    "result": "sausage_blood_raw",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "id_suffix": "large",
+    "difficulty": 4,
+    "charges": 4,
+    "time": "10 m",
+    "autolearn": true,
```
Comment (Lamandus) [inline data/json/recipes/food/meat.json:1803]:
```suggestion
    "autolearn": true,
    "book_learn": [ [ "family_cookbook", 2 ], [ "scots_cookbook", 2 ] ],
```

File: data/json/recipes/food/meat.json [hunk 7]
```diff
@@ -1691,6 +1841,20 @@
...
+    "result": "sausage_blood_cooked",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_MEAT",
+    "skill_used": "cooking",
+    "time": "30 m",
+    "autolearn": true,
+    "batch_time_factors": [ 80, 1 ],
+    "qualities": [ { "id": "COOK", "level": 1 } ],
+    "tools": [ [ [ "surface_heat", 2, "LIST" ] ] ],
+    "charges": 1,
```
Comment (Lamandus) [inline data/json/recipes/food/meat.json:1855]:
```suggestion
```

**Review by Lamandus — CHANGES_REQUESTED** [review]

**Comment by Lamandus** [comment]
consider adding spawns into the game for those sausages.

if you use notepad++, check the folder "itemgroups" and search for the item "sausage" (and their variants). Add the blutwurst to it with around 1/4 of the chance of spawning it.

**Comment by Lamandus** [comment]
https://en.wikipedia.org/wiki/Schwarzsauer

https://en.wikipedia.org/wiki/Blood_soup

are also things to consider

**Comment by Lamandus** [comment]
https://github.com/cataclysmbnteam/Cataclysm-BN/pull/4762
as a follow up.

**Comment by Lamandus** [comment]
please update this branch from main to rerun the tests and put it "away" from draft.

**Comment by chaosvolt** [comment]
Went ahead and updated it from upstream for you along with fixing a spacing thing it complained about, so should be good to mark as ready.

**Review by chaosvolt — APPROVED** [review]

**Pull request merged**